src/Process/SkipLintProcess.php

```diff
@@ -23,15 +23,12 @@ class SkipLintProcess extends PhpProcess
     public function __construct(PhpExecutable $phpExecutable, array $filesToCheck)
     {
         $scriptPath = __DIR__ . '/../../bin/skip-linting.php';
-        $script = file_get_contents($scriptPath);
 
-        if (!$script) {
+        if (!is_file($scriptPath)) {
```

Comment:
```suggestion
        if (!@is_file($scriptPath)) {
```

I think it would make sense to silence a [potential warning coming from `is_file()`](https://www.php.net/manual/en/function.is-file.php#refsect1-function.is-file-errors) as the exception being thrown makes the warning redundant.